eldev-util.el

```diff
@@ -218,6 +218,84 @@ parameter, but it's not needed in noninteractive use."
     (if (> (length value) 0) value if-empty-or-not-set)))
 
 
+(defun eldev-bat-quote (mode &optional string)
+  "Helper function to quote data in a batch file using Delayed
+  Expansion. The :init MODE result has to be inserted first in a
+  batch file to prepare the ground for the other MODEs to work.
+
+  Available MODEs:
+
+  :init
+
+    Returns the prolog required to support the rest of the quoting
+    modes. Basically this sets the ARGS variable to the batch
+    command line arguments, enables delayed expansion and creates
+    the NL variable containing the newline character.
+
+  :args
+
+    Returns the delayed expansion reference to the ARGS variable
+    which holds the command line arguments of the batch file.
+
+  :string
+
+    Returns the quoted STRING using delayed expansion tricks to
+    support multiline text.
```

Comment:
Just for consistency, remove the final newline in the docstring.

Reply:
Done.